README.md

```diff
@@ -21,6 +21,8 @@ Happy scripting with Amber!
 ```sh
 # Install dependencies
 npm install
+# Generate the stdlib documentation
+./sync-stdlib-doc.sh
```

Comment:
@Mte90 can we change this to Amber file?

```suggestion
./sync-stdlib-doc.ab
```

Reply:
ops